CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Smart Collections - Add a field parameter to allow the sorting on it.
```

Comment:
`Smart Fields - Add a parameter to specify if the sorting is allowed on this field.`